config.lua

```diff
@@ -32,33 +32,5 @@ QBConfig.Server.Permissions = { 'god', 'admin', 'mod' } -- Add as many groups as
 QBConfig.Notify = {}
 
 QBConfig.Notify.NotificationStyling = {
-    group = true, -- Allow notifications to stack with a badge instead of repeating
-    position = "right", -- top-left | top-right | bottom-left | bottom-right | top | bottom | left | right | center
-    progress = true -- Display Progress Bar
-}
-
--- These are how you define different notification variants
--- The "color" key is background of the notification
--- The "icon" key is the css-icon code, this project uses `Material Icons` & `Font Awesome`
-QBConfig.Notify.VariantDefinitions = {
-    success = {
-        classes = 'success',
-        icon = 'done'
-    },
-    primary = {
-        classes = 'primary',
-        icon = 'info'
-    },
-    error = {
-        classes = 'error',
-        icon = 'dangerous'
-    },
-    police = {
-        classes = 'police',
-        icon = 'local_police'
-    },
-    ambulance = {
-        classes = 'ambulance',
-        icon = 'fas fa-ambulance'
-    }
+    position = 'top-right', -- 'top' | 'top-right' | 'top-left' | 'bottom' | 'bottom-right' | 'bottom-left'
 }
```

Comment:
Couldn't this be trimmed further towards something like
```
QBConfig.Notify.Position = 'top-right'
```

Reply:
Yeah, it could be. I wasn't sure if I wanted to change the original structure. What do you think?

Reply:
Probably better since now you are just outputting an array/table containing one value. Especially since ox_lib will just take a string as an argument.

The only thing to watch out for would be any bits of code still using the old notify system.

Reply:
Simplified to just a string.